main/config.py

```diff
@@ -13,28 +13,32 @@
 
   APPLICATION_ID = app_identity.get_application_id()
 except (ImportError, AttributeError):
-  pass
-else:
-  from datetime import datetime
+  APPLICATION_ID = "Testing"
 
-  CURRENT_VERSION_ID = os.environ.get('CURRENT_VERSION_ID')
-  CURRENT_VERSION_NAME = CURRENT_VERSION_ID.split('.')[0]
-  CURRENT_VERSION_TIMESTAMP = long(CURRENT_VERSION_ID.split('.')[1]) >> 28
-  if DEVELOPMENT:
-    import calendar
+from datetime import datetime
+
+CURRENT_VERSION_ID = os.environ.get('CURRENT_VERSION_ID') or "Testing.0"
+CURRENT_VERSION_NAME = CURRENT_VERSION_ID.split('.')[0]
+if DEVELOPMENT:
+  import calendar
 
-    CURRENT_VERSION_TIMESTAMP = calendar.timegm(datetime.utcnow().timetuple())
-  CURRENT_VERSION_DATE = datetime.utcfromtimestamp(CURRENT_VERSION_TIMESTAMP)
-  USER_AGENT = '%s/%s' % (APPLICATION_ID, CURRENT_VERSION_ID)
+  CURRENT_VERSION_TIMESTAMP = calendar.timegm(datetime.utcnow().timetuple())
+else:
+  CURRENT_VERSION_TIMESTAMP = long(CURRENT_VERSION_ID.split('.')[1]) >> 28
+CURRENT_VERSION_DATE = datetime.utcfromtimestamp(CURRENT_VERSION_TIMESTAMP)
+USER_AGENT = '%s/%s' % (APPLICATION_ID, CURRENT_VERSION_ID)
 
-  import model
+import model
 
+try:
   CONFIG_DB = model.Config.get_master_db()
   SECRET_KEY = CONFIG_DB.flask_secret_key.encode('ascii')
   RECAPTCHA_PUBLIC_KEY = CONFIG_DB.recaptcha_public_key
   RECAPTCHA_PRIVATE_KEY = CONFIG_DB.recaptcha_private_key
-  RECAPTCHA_LIMIT = 8
   TRUSTED_HOSTS = CONFIG_DB.trusted_hosts
+except AssertionError:
+  CONFIG_DB = model.Config()
+RECAPTCHA_LIMIT = 8
```

Comment:
let's move that with the rest of the group

Reply:
I think it needs to be set whether or not the other RECAPTCHA variables are used doesn't it? (Because of how `form_with_recaptcha` is defined.)